src/mono/sample/iOS-NativeAOT/Program.csproj

```diff
@@ -49,7 +48,7 @@
       <PropertyGroup>
         <IlcPath>$([MSBuild]::NormalizeDirectory('$(RepoRoot)', 'artifacts', 'bin', 'coreclr', '$(HostOS).$(BuildArchitecture).$(CoreCLRConfiguration)', 'ilc'))</IlcPath>
         <IlcToolsPath>$(IlcPath)</IlcToolsPath>
-        <IlcSdkPath>$(CoreCLRAotSdkDir)</IlcSdkPath>
+        <IlcSdkPath>$([MSBuild]::NormalizeDirectory('$(RepoRoot)', 'artifacts', 'bin', 'coreclr', '$(TargetOS).$(TargetArchitecture).$(CoreCLRConfiguration)', 'aotsdk'))</IlcSdkPath>
```

Comment:
Why is this necessary?
If I remember correctly `CoreCLRAotSdkDir` is set in `liveBuilds.targets` and already matches the change you made. Maybe it gets overwritten in the process?

Reply:
I've confirmed that the `IlcSdkPath` can use the `CoreCLRAotSdkDir` property.

Reply:
Update: The `CoreCLRAotSdkDir` relies on the `CoreCLRArtifactsPath` property which is set to the host `os-arch` as it is used for the ILC path resolution.